algosdk/future/transaction.py

```diff
@@ -477,6 +483,161 @@ def __eq__(self, other):
                 self.votekd == other.votekd)
 
 
+class OnlineKeyregTxn(Transaction):
```

Comment:
This class (and `OfflineKeyregTxn`) should extend `KeyregTxn`.

Reply:
Fixed.